src/index.css

```
@@ -157,6 +157,19 @@ a:focus {
 	display: none;
 }
 
+.button {
+
+	padding: 25px 30px;
+	border-radius: 40px;
+	background-color: #ff7a7a;
+	color: #4e4e4e;
+	font-size: 14px;
+	font-family: soleil, sans-serif;
```

The font doesn't match the gray text. I grabbed this from **.content p** in index.css and it visually matches. Could you replace  font-family: soleil, sans-serif; with what's below? Sorry, the CSS I copied is using [CSS Variables](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties).

```suggestion
	font-family: var(--font-alt);
```